python/python/tests/test_fts.py

```diff
@@ -324,11 +324,18 @@ def test_search_fts_phrase_query(table):
         pass
     table.create_fts_index("text", use_tantivy=False, with_position=True, replace=True)
     results = table.search("puppy").limit(100).to_list()
-    phrase_results = table.search('"puppy runs"').limit(100).to_list()
+
+    # Test with quotation marks
+    phrase_results = table.search("puppy runs").phrase_query().limit(100).to_list()
```

Comment:
We should keep this like so, to match the comment, right?
```suggestion
    phrase_results = table.search('"puppy runs"').limit(100).to_list()
```

Reply:
🤦 yeah whoops